yt/frontends/adaptahop/data_structures.py

```diff
@@ -23,6 +22,7 @@
 from yt.units import Mpc
 from yt.utilities.cython_fortran_utils import FortranFile
 
+from ...data_objects.selection_objects.base_objects import YTSelectionContainer
```

Comment:
wait this is valid python??

Reply:
Yeah, but I actually am going to change these; I would rather absolute imports than relative for these.  I missed that pycharm did this.  (Yeah, that's right, I used a tool to help me refactor!)

Reply:
>  (Yeah, that's right, I used a tool to help me refactor!)
*screams in vanilla emacs*

Reply:
Well I *tried* to get `rope` to work but couldn't!